test/integration/local_docker_registry.py

```diff
@@ -0,0 +1,47 @@
+import docker
+import contextlib
+import json
+import logging
+import requests
+import time
+
+from typing import Dict, List
+from test.ports import find_free_port
+from exasol.ds.sandbox.lib.dss_docker import DssDockerImage, DockerRegistry
+
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.DEBUG)
+
+
+class LocalDockerRegistry(DockerRegistry):
```

Comment:
Compared to [ITDE](https://github.com/exasol/integration-test-docker-environment/blob/main/exasol_integration_test_docker_environment/testing/docker_registry.py) I did not need a `LocalDockerRegistryContextManager` as the pytest fixture already provides context-like functionality.

The reason for the earlier construct returning a context was the parameter `repository` to the registry which now has become obsolete and has been removed.